person_class.rb

```diff
@@ -5,7 +5,7 @@
 class Person < Nameable
   # Getters&setters for @id, @name, and @age
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
```

Comment:
- [ ] [OPTIONAL] Good job adding attribute accessor for rentals, however please also initialise it so that it works properly currently we get an error when we try to add rental to a person please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/224077786-d5d0130d-31d4-489c-bd23-cce7e95ffb08.png)

you will need to initialise` @rentals = []` for it to work correctly